changelogs/current.yaml

```diff
@@ -364,6 +364,11 @@ new_features:
 - area: access_log
   change: |
     Added support for ``%CONNECTION_ID%`` command operator for UDP session access log.
+- area: tls
+  change: |
+    Added new metric for emitting seconds since UNIX epoch of expirations of TLS & CA certificates.
+    They are rooted at cluster.<cluster_name>.ssl.certificate.<cert_name>.
+    and at listener.<address>.ssl.certificate.<cert_name>. namespace.
```

Comment:
```suggestion
    They are rooted at ``cluster.<cluster_name>.ssl.certificate.<cert_name>.``
    and at ``listener.<address>.ssl.certificate.<cert_name>.`` namespace.
```

Reply:
looks like this isnt resolved in current version - also this seems to be deleting unrelated change logs

Reply:
thanks updated. Must have accidentally removed it on resolving main merge conflict.